ros_package_template/src/template_ros_node.cpp

```diff
@@ -0,0 +1,26 @@
+#include <ros_package_template/template_ros.hpp>
+#include <ros_package_template/template_ros_pub.hpp>
+
+using namespace ariitk::ros_package_template;
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "template_node");
+    ros::NodeHandle nh;
+    
+    TemplateROS fitter;
```

Comment:
@gajena Instead of writing 4 lines declaring the objects and then initializing it using a seperate init function, why not write this in the constructor? The constructor can take the nodehandle as the argument.

Reply:
This will eliminate the need to define a seperate init function.